playground/static/main.js

```diff
@@ -158,32 +158,56 @@ function toggleView() {
 }
 
 async function loadPackageInfo() {
-    const info = JSON.parse(await compilerAction({type: "package_info"}));
+    const info = JSON.parse(await compilerAction({ type: "package_info" }));
 
-    const type_str = (type) => {
+    const type_annotation = (type) => {
+        let type_name = type;
+        let color = "";
         if (Array.isArray(type)) {
-            return type.join("/");
+            type_name = type.join("/");
+            color = "#90A959";
+        } else if (type == "String") {
+            color = "#2A6041";
+        } else if (type == "Unsigned integer") {
```

Comment:
Suggestion: Disable type coercion
```suggestion
        } else if (type === "Unsigned integer") {
```